pkg/kubernetes/deploy.js

```diff
@@ -21,15 +21,22 @@ define([
     "jquery",
     "base1/cockpit",
     "kubernetes/client",
-    "base1/mustache",
-    "base1/patterns"
-], function($, cockpit, kubernetes, Mustache) {
+    "kubernetes/nulecule",
+    "base1/mustache"
+], function($, cockpit, kubernetes, nulecule, Mustache) {
     "use strict";
 
     var _ = cockpit.gettext;
 
     /* The kubernetes client: valid while dialog is open */
     var client;
+    /* The Nulecule client: valid while dialog is open */
+    var nulecule_client;
+
+    var run_stage = false;
+    var answerfile = {};
+    var install_dir = "";
```

Comment:
Why is this a global? Shouldn't the nulecule_client hold this value as a member property?